src/main/webapp/search_area.js

```diff
@@ -75,6 +75,7 @@ class SearchArea {
     this.filterInputArea.addEventListener('keypress', (e) => {
       if (e.key === 'Enter') {
```

Comment:
Where does the user hit enter? Is there a specific enter button?

Reply:
This is a keypress for the inputArea, whenever "Enter" is hit in that input area, the input is sent